content/guides/depin/getting-started.md

```diff
@@ -0,0 +1,208 @@
+oriented to these common on-chain DePIN use cases. Topics covered include:
+
+- Minting and Managing a Token
+- Rewards Distribution
+- Proof of Contribution
+- Governance
+
+## Minting and Managing a Token
+
+### Token Minting
+
+When minting a token on Solana, there are two token programs to choose from:
+[the Token program](https://spl.solana.com/token) or
+[the Token22 program](https://spl.solana.com/token-2022). There are tradeoffs to
+consider (discussed
+[here](https://solana.stackexchange.com/questions/9205/what-is-the-advantage-of-using-the-token22-token-extensions-program-over-the-old)).
+The recommended selection between the two options ultimately reduces to whether
+the features in the token extensions program would be of use to the application.
+
+### Token Listing
+
+In order to have your token appear in explorers, decentralized exchanges, and
+other ecosystem token lists,
+[getting the token verified through Jupiter Verification](https://station.jup.ag/guides/general/get-your-token-on-jupiter)
+is recommended.
+
+### Modeling Token Distribution
+
+For DePIN teams looking to optimize the way they allocate their token across
+various needs and stakeholders (rewards for participating / supplying network
+resources, private or public token sales etc.), tools like
+[Forgd](https://www.forgd.com/) can be used to model out and refine token
+distribution strategies.
+
+### Token Management
+
+Most teams utilize [Squads](https://squads.so/) for their on-chain treasury
+management, also leveraging their multisig feature.
+
+## Rewards Distribution
+
+There are two common paths for rewards distributions: Merkle claims and ZK
+compression. Each of these optimize the cost of distributing a potentially small
+value of tokens to a large set of users by allowing the application to do so in
+a single transaction per claim period.
+
+### Merkle Claims
+
+Rewards can be distributed using a Merkle tree structure, allowing for efficient
+batch processing of claims. This approach is used to minimize the number of
+transactions on the blockchain by allowing users to claim their rewards based on
+a published Merkle root.
+
+The application constructs a Merkle tree on a regular basis and publishes the
+root on-chain. Each leaf node represents a recipient’s rewards.
+
+In order for users to claim their rewards, they generate a Merkle proof that
+demonstrates a particular leaf is part of the published Merkle root. Once their
+claim is verified, the rewards are distributed to the user’s wallet.
+
+See
+[example code](https://gist.github.com/lanvidr/88a594da06ba867bf8201fe8c6331dc0)
+and
+[Jupiter’s Merkle distributor](https://github.com/jup-ag/merkle-distributor-sdk)
+for an additional reference.
+
+#### Automatic Distribution
+
+An alternative to having users proactively claim rewards is automating rewards
+distribution. On a regular basis, this “rewards crank” fetches rewards for an
```

Comment:
I don't have any links for this. We can also change this language to call it a scheduled task instead of a crank, because that term sounds more familiar to me

Reply:
cranks are common, i think people are familiar enough with it. I just read the paragraph as might be trying to point to a link of something and wanted to make sure we are not missing a link

Reply:
I don't have a link for this, but feel free to change wording via the suggestion feature